bobanetwork/gateway
# 🧹 Cleanup | Fixes | Updates | Missing Parts | Sentry Error

:clipboard:  

closes: https://github.com/bobanetwork/gateway/issues/454
closes: https://github.com/bobanetwork/gateway/issues/70
closes: https://github.com/bobanetwork/gateway/issues/452
closes: https://github.com/bobanetwork/gateway/issues/451


Cleaning Dead Code:
---

- unused signature reducer
- unused signature action
- clean network service functions
- code linting & formating
- rename variable name
- cleanup settle actions along with selectors
- coingecko to load url from const.
- removed transfer action & service
- cleanup comment from daoAction
- clean up unused constants
- unused graphql file & test
- unused masterconfig file
- unused transaction action
- updated gateway version in test to v0.14.4
- updated package @bobanetwork/register @babel/preset-typescript
- updated broken snapshots for test

Light Bridge Updates:
---
- Clean up fast bridge tooltip info
- Added tooltip with light bridge info.
![Screenshot 2024-04-04 at 12 13 47 AM](https://github.com/bobanetwork/gateway/assets/86316370/61bdb12b-7a92-4f50-83e8-cf32d76bf36e)


Emergency Swap :
--- 
- enable emergency swap for BNB L2 network only
- extended the useNetworkInfo with check for activeNetworkBNB
- Returning true from service.
- clean up Ethereum specific logic from component and service functions.
- updated getETHMetaTransaction name  => swapToken
- increase scope of try catch block in swap function
- removed ethereum based check and logics.

Switch Fee Dropdown : 
---
- enable select token fee switcher for BNB L2 only
- In case BNB L1 showing tooltip with question
- disable fee switcher for ETH L2
- rename function from switchFee -> switchFeeToken.

Sentry Error Fixes:
---
- toLowerCase unable to find on undefined [issue link](https://enya-inc.sentry.io/issues/5092587526/?query=is%3Aunresolved&referrer=issue-stream&sort=freq&statsPeriod=14d&stream_index=24)
- Removed unused capturing of low boba balance for swap [issues link](https://enya-inc.sentry.io/issues/3598941538/?project=6727751&query=is%3Aunresolved&referrer=issue-stream&sort=freq&statsPeriod=14d&stream_index=5)
- destructure property crash on routes [issues link](https://enya-inc.sentry.io/issues/5137865843/?project=6727751&query=&referrer=project-issue-stream) [issue link 1](https://enya-inc.sentry.io/issues/5067543744/?project=6727751&query=&referrer=project-issue-stream) [issue link 2](https://enya-inc.sentry.io/issues/5078544530/?project=6727751&query=&referrer=project-issue-stream)
- fix:Cannot read properties of null (reading 'toLowerCase') [issue link](https://enya-inc.sentry.io/issues/5099376224/?project=6727751&query=&referrer=project-issue-stream)

Mobile View Fixes:
---
- Notification banner height in case of mobile
- Copy address in case of mobile
- Disable network selector in case of mobile
- Increase width of bridging case of mobile
